content/md/assets/full-automatic-pam-migration.md

```diff
@@ -0,0 +1,112 @@
+In this migration, **everything is automated**, you just need to launch one single command and your PAM assets will be migrated in the 4.0 new Asset Manager:
+- One asset family will be created gathering all you assets.
+- All your assets will be migrated and linked to the new asset family.
+- All you existing PAM asset collection product attributes will be linked to the new asset family.
+- And all your links between assets and products / product models will be kept.
+
+# What will my asset family gonna look like?
+The asset family created will contain the following attributes:
+- `reference` (*media_file attribute*): containing the reference file of the PAM asset and used as attribute as main media for this family
+- `reference_localizable` (*media_file attribute*): an attribute with a value per locale containing the reference file of the asset for each locale, if the asset was localizable in the PAM
+- `variation_scopable` (*media_file attribute*): an attribute with a value per channel containing the asset variation file for each channel
+- `variation_scopable_localizable` (*media_file attribute*): an attribute with a value per channel containing the asset variation file for each channel and locale, if the asset was localizable in the PAM
+- `description` (*text attribute*):  with the PAM asset description
+- `categories` (*text attribute*): with the PAM asset categories separated by a comma
+- `tags` (*text attribute*): with the PAM asset tags (tags are separated by a comma)
+- `end_of_used`: text attribute with the asset end of use date
+
+During the migration, you will be able to choose the asset family code.
+
+![PAM assets family](pam-assets-family.png)
+
+# What will my assets gonna look like?
+
+![PAM asset](pam-asset.png)
+
+# What are the steps to migrate?
+Don't be afraid, this part is a bit technical. If you are a Julia, you can ask your IT guy or your integrator to do it.
+
+4 steps to migrate your PAM assets:
+1. Install & Setup the migration tool
+1. Launch the PAM assets migration
+1. Check and test the assets migration
+1. Set transformations (*optional*, only if you use real transformations in the PAM)
+
+
+## _Step 1_ | Install & Setup the migration tool
+We built a dedicated tool [CSVToAsset](https://github.com/akeneo/CsvToAsset) to migrate your PAM assets to new Asset Manager assets.
+
+To install and setup this tool, refer to its [readme file](https://github.com/akeneo/CsvToAsset/blob/master/README.md).
+
+## _Step 2_ | Launch the PAM assets migration
+Then, you will launch one single command of the [CSVToAsset](https://github.com/akeneo/CsvToAsset) tool and your PAM assets will be migrated in the 4.0 new Asset Manager.
+
+Run `./bin/migrate.php <family-code> <path-to-ee-installation>` with the family-code you choose for the asset family created.
+
+This command will:
+1. Export the PAM assets in CSV into a temporary folder
+1. Create dedicated API credentials
+1. Migrate the assets data:
```

Comment:
Maybe for this step, I would just say that, it creates the asset family and imports the exported assets in the asset manager will be enough no ? I mean they don't need to know much more if they use the full-auto mode? 

What do you think ?

Reply:
oui, je peux donner moins de détails